### PR TITLE
Add in city data from Teleport API, and accompanying helper function

### PR DESCRIPTION
**Pull request by rachelpr**
Added in city facts data and QoL data, minus description for now. 

Helper function is to cut down the amount of QoL tiles from 17 to the desired 9

### CODE REVIEW

**Comment by MGS31** [comment]
Looks solid to me! I'll let andrea review as well and approve before we pull as it's still set to 1 user only for merging lol

**Comment by b1u3too** [comment]
[Idea for the html tags in the description reponse?](https://www.npmjs.com/package/string-strip-html)
(they look to be an OG styling trick, with those <b> tags on words and phrases they'd like bolded)

File: leafsmart-front-end/src/components/CityFacts.js
Comment (b1u3too) [inline leafsmart-front-end/src/components/CityFacts.js:9]: Were we going to use axios instead of fetch?

File: leafsmart-front-end/src/components/QoLData.js
Comment (b1u3too) [inline leafsmart-front-end/src/components/QoLData.js:115]: Freakin' LURVE this. I left a comment on Matt's about List/ListItem naming conventions. Let's talk about what we'd like to do so that we can make it consistent between the two!

**Review by b1u3too — COMMENTED** [review]
Looks rad! Resisting the urge to push through an approval so we can talk about those naming conventions before merging!

File: leafsmart-front-end/src/components/CityFacts.js
Comment (rachelpr) [reply leafsmart-front-end/src/components/CityFacts.js:9]: yep, forgot to fix it in the CityFacts

**Review by b1u3too — APPROVED** [review]
Magnifico!

**Review by MGS31 — APPROVED** [review]
This looks great!

**Pull request merged**
